.gitignore

```diff
@@ -11,10 +11,12 @@ vchat.db*
 *.before
 *.pyc
 *.pid
-data
-data/
 cfg/
 
+#Ignore everything in datafolder and subdirectories
+/data/**/*
+/tmp/**/*
+
```

Comment:
Removing the `data` line will cause the `data` symlink to be deleted.   The lines being added in as replacements only ignore the _contents_ of data, not data itself.